server/data/helpers/refreshrHelper.js

```diff
@@ -6,8 +6,9 @@ module.exports = {
 
     return allRefreshrs;
   },
+  getClassRefreshrs: async class_id => db('refreshrs').where({ class_id }),
 
```

Comment:
Cool, so this will pull refreshrs based on class ID and then we would do an axios call on the front end to display them when a teacher clicks on a class?

Reply:
Yep! We may or may not need this one in the end because the getClass helper has this information. But for the page Tim and I were working on yesterday we really only need the title and date of the refreshr and my brain was too broken to try and restructure the shape of getClass. :slightly_smiling_face: